### ISSUE TITLE
Mobs should be able to reset combat if they evade for too long in instances

### ISSUE THREAD

**Issue by ohganot** [LightsHope/server#54]
**Issue**
Mobs never drop combat in instances if they go into evade mode.

**How it should be**
When you have aggro on a mob in instance and move into a spot the mob cant reach, then the mob should go into evade mode and drop combat if you stay there for long enough (10-15 seconds).

**Proof**
https://www.youtube.com/watch?v=u-2u48dD2EI (The fire elementals before Incendius drop combat after 10-15 seconds after they go into evade mode by jumping down the bridge)

On Light's Hope, the fire elementals will bug out and follow you through the air instead of evading and resetting.

This also causes  problems with the fish in Zul'gurub since they aren't resetting combat when you go outside water where they cant follow you.

**Comment by brotalnia** [comment]
You seem to be correct. I like it when people actually provide proof.

**Comment by ohganot** [comment]
Sometimes proof is hard to find if the issue is very specific. But yeah.

This should apply to all instances and raids, not only ZG and BRD.

**Comment by ohganot** [comment]
Another clip where mobs reset after being in evade mode/unable to reach the player for 15 seconds:
https://www.youtube.com/watch?v=kRXGQpMeKi0 1:50-2:20

**Comment by dimefox** [comment]
Also #810 

There's currently a 24s timer for non-instance mobs to evade. It's simple enough to apply to everything but looks intentional. I see in Trinity they do allow it for dungeons but not raids.

Is there any evidence that evade timer happens for raid trash too?

**Comment by Cabro** [comment]
Not all mobs should happen like this. Remember vanilla tactics for Mandokir? With the stairs location so Ohgan CAN'T REACH THE RAID?

**Comment by ohganot** [comment]
Yeah, its always different for bosses, they generally don't reset unless you wipe or they have a specifically coded leash range.
